src/proxy-ui-api/frontend/src/views/AppLogin.vue

```diff
@@ -184,6 +185,14 @@ export default (Vue as VueConstructor<
           this.$bus.$emit('show-error', error.message);
         });
     },
+    async fetchSecurityServerVersion() {
+      this.$store
+        .dispatch('fetchSecurityServerVersion')
+        .catch((error) => {
+          console.error(error);
+          this.$bus.$emit('show-error', error.message);
```

Comment:
I'd say logging error is not needed here either as if control reaches catch-block there will be notification for the user and the error can be found from the network activity. Also this logs (nearly) the same error as in the previous comment

Reply:
I think it is still good practice to log it into the console as well because that is a more obvious place to look for it rather than finding the correct request under the network tab. 

I removed the console log in this component to prevent duplication but kept it in the store level.